src/components/Layout/DashBoardLayout/index.tsx

```diff
@@ -1,20 +1,36 @@
 import DashBoardHeader from '@/components/Header/DashBoard';
 import SideMenu from '@/components/SideMenu';
-import { atom } from 'jotai';
-import { PropsWithChildren } from 'react';
+import HttpClient from '@/apis/httpClient';
+import instance from '@/apis/axios';
+import { PropsWithChildren, useEffect } from 'react';
+import { dashboardsAtom } from '@/utils/jotai';
+import { useAtom } from 'jotai';
 
-export const sideMenuAtom = atom(false);
-
-// 테스트 데이터
-const dashboards = [
-  { id: 1, title: '송은', color: '#7AC555' },
-  { id: 2, title: '유빈', color: '#760DDE' },
-  { id: 3, title: '승구', color: '#FFA500' },
-  { id: 4, title: '우혁', color: '#76A5EA' },
-  { id: 5, title: '봉찬', color: '#E876EA' },
-];
+type DashBoard = {
+  id: number;
+  title: string;
+  color: string;
+  createdAt: string;
+  updatedAt: string;
+  createdByMe: boolean;
+  userId: number;
+};
 
 function DashBoardLayout({ children }: PropsWithChildren) {
+  const [dashboards, setDashboards] = useAtom(dashboardsAtom);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const httpClient = new HttpClient(instance);
+      const dashboardsData = await httpClient.get<{ dashboards: DashBoard[] }>(
+        '/dashboards?navigationMethod=infiniteScroll',
+      );
+      setDashboards(dashboardsData.dashboards);
+    };
+
+    fetchData();
+  }, []);
+
```

Comment:
```suggestion
type DashBoard = {
  id: number;
  title: string;
  color: string;
  createdAt: string;
  updatedAt: string;
  createdByMe: boolean;
  userId: number;
};

function DashBoardLayout({ children }: PropsWithChildren) {
  const [dashboards, setDashboards] = useAtom(dashboardsAtom);

  useEffect(() => {
    const fetchData = async () => {
      const httpClient = new HttpClient(instance);
      const dashboardsData = await httpClient.get<{ dashboards: DashBoard[] }>(
        '/dashboards?navigationMethod=infiniteScroll',
      );
      setDashboards(dashboardsData.dashboards);
    };

    fetchData();
  }, []);

```
여기 부분은 제가 이미 해서 PR 올린 부분이라서 지워주셔도 좋을 것 같아요